### ISSUE TITLE
Comparison mode to compare different model outputs

### ISSUE THREAD

**Issue by centminmod** [enricoros/big-agi#126]
Now we have so many OpenAI models to choose from, would love it if we could optionally compare models i.e. GPT3.5 vs GPT 4 etc with interface like at https://play.vercel.ai :)

**Comment by enricoros** [comment]
Good idea. The question is how to integrate this with the chat (history of messages). Would we split the screen, try 2 models, and then pick left or right, and delete the other?

**Comment by centminmod** [comment]
Not entirely sure myself. Maybe ChatGPT has a clue 😆

**Comment by centminmod** [comment]
> Would we split the screen, try 2 models, and then pick left or right, and delete the other?

Was thinking about this some more. Maybe after both left and right outputs are shown, give use option to export to HTML and other formats the 2 side by side outputs and then choose one for saving to chat history :)

**Comment by enricoros** [comment]
Also see #208

**Comment by enricoros** [comment]
@hourianto good point, working on this.

**Comment by enricoros** [comment]
Closing as shipped.

Beam is this feature and has been since 1.16: pick N models, run them on the same turn side by side, then Compare or Fuse them, copy them all out, or send one straight into the chat. @hourianto, the question you raised in this thread is the part Beam does not answer and that I said I was working on - split panes each on their own model. It is still not in the open-source build, where every pane resolves the one app-level chat model, and it now has a proper home in #1173, which pins the model to the conversation and shows a per-pane badge. I have carried it there. @centminmod, the other loose end from your comment, exporting the side-by-side outputs to HTML before picking one, was never built; Beam's Copy gives you every response as text separated by ---.

@centminmod this close comes from an automated backlog pass - if you disagree, feel free to reopen.
